lib/highlight.ts

```diff
@@ -16,11 +16,14 @@ interface Options {
   mark?: number[];
   tab?: string;
   wrap?: boolean;
+  disableStripIndent?: boolean
```

Comment:
IMHO, I prefer to use `stripIndent` instead of `disableStripIndent` for this naming. Because, other existing boolean options are positive form, but this one is negative form.